manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -156,7 +156,7 @@ spec:
           {{- if .Values.global.caAddress }}
             value: {{ .Values.global.caAddress }}
           {{- else }}
-            value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Values.global.istioNamespace }}.svc:15012
```

Comment:
That's where it gets a bit tricky - AFAIK the intent (and best practice) is to run egress/ingress gateways in a separate namespace - but still be able to point to a custom namespace.

I don't know if defaulting to the less secure mode ( egress/ingress in same namespace with istiod and the root CA certs) is a good idea. I would remove this for now ( in particular since we're recommending injected or K8S Gateway mechanisms ).

Reply:
Because I haven't actually removed the default value of `istio-system` ... I am not actually changing the default behavior.